infra/src/main/kotlin/fi/fta/geoviite/infra/linking/Publication.kt

```diff
@@ -15,6 +15,7 @@ open class Publication(
     open val id: IntId<Publication>,
     open val publicationTime: Instant,
     open val publicationUser: UserName,
+    open val message: String?,
```

Comment:
Nullable, jotta vanhat julkaisut eivät hajoisi. Null vaikutti fiksummalta defaulttiarvolta niille kuin esim. tyhjä merkkijono.

Reply:
Jäin vielä miettii, että pitäisikö tuo pakottaa not nulliksi ja vanhoille julkaisuille ajettaisiin tyhjä stringi db-migraatiolla.

Reply:
Tämä on ihan hyvä pointti. Nyt taitaa olla mahdollista antaa sekä null että ""? Tuo ei sinällään ole järkevää koska se on 1 tarpeeton mahdollinen tila. Sinällään tässä on useampi vaihtoehto jolla korjata - tuo "" nullin sijaan on simppeli, tosin ratkaisee vain osan ongelmaa. 

Omasta mielestä ehkä paras on, että jos nuo on tarkoitus olla jatkossa aina, niin tähän kannattaisi lisätä se sanitoitu tyyppi (mainittu alla) ja siellä varmistaa myös että pituus on jotain järkevää, jolloin "" ei ole enää validi arvo. Tuossa kohtaa vanhoille julkaisuille ainoa fiksu vaihtoehto lienee null, ellei sitten laiteta jotain vakiostringiä niihin kaikkiin (mikä tuntuu vähän meh-ratkaisulta). Uuden julkaisun luonnissa on helppo lisätä tuo samainen sanitoitu tyyppi publish-requestiin non-nullina, jolloin nulleja ei voi tehdä lisää.